core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -21,13 +21,26 @@
  */
 public class ClientConfigProperties extends AbstractConfig {
 
+    /**
+     * The default amount of time to wait for credits after link creation.
+     */
+    public static final long DEFAULT_FLOW_LATENCY = 20L; //ms
+    /**
+     * The default number of credits issued by the receiver side of a link.
+     */
+    public static final int  DEFAULT_INITIAL_CREDITS = 200;
```

Comment:
Did you changed this on purpose? If yes, do think 200 is a better value?

Reply:
I thought 200 would be a little more *defensive* and *cautious*. I also have been inspired by the 250 credits that the dispatch router hands out by default (and which seem to be enough to be able to process 30000 msg/s).